plugin/pom.xml

```diff
@@ -76,6 +76,16 @@
 
     <dependency>
       <groupId>io.fabric8</groupId>
+      <artifactId>fabric8-maven-enricher-karaf</artifactId>
```

Comment:
Let's start a `fabric8-maven-enricher-healthcheck` module and see how it feels. I agreed its easier to find spring-boot related code when the modules are called like that, on the other hand one could argue, when looking for healt-check it's easier when there is a heatlcheck module. It all depends on the perspective ;-)

We can still later move or split stuff. E.g. when we decide to have the tech as main focus, we could also combine the spring-boot generator, the spring-boot healthcheck and maybe later the spring-boot "watcher" into a dedicated 'spring-boot' module.